backend/src/Request/Subscription/SubscriptionHistoryCreateRequest.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace App\Request\Subscription;
+
+class SubscriptionHistoryCreateRequest
+{
+    private $storeOwner;
+
+    private $note;
+
+    private $type;
+
+    private $createdAt;
```

Comment:
I guess we do not need to define createdAt in the request, because we already set it in the manager layer using the entity not the request.

Reply:
done